### PR TITLE
spec for Settlements, Adjustments and Reimbursements APIs

### PR DESCRIPTION
**Pull request by sanskarsharma**

### CODE REVIEW

File: src/admin/openapi.yaml [hunk 1]
```diff
@@ -120,11 +123,13 @@ paths:
   /admin/advances:
     $ref: paths/admin@advances.yaml
   /admin/settlements:
-    $ref: paths/admin@settlements.yaml
+   $ref: paths/admin@settlements.yaml
```
Comment (jayachandhar) [inline src/admin/openapi.yaml:126]: fix indentation

File: src/components/schemas/adjustment.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    
+    org_id:
+      $ref: './fields.yaml#/org_id'
+    user_id:
+      $ref: './fields.yaml#/user_id'
+    issuer_user_id:
+      $ref: './fields.yaml#/user_id'
+
+    from_account_id:
+      $ref: './fields.yaml#/id_string'
```
Comment (jayachandhar) [inline src/components/schemas/adjustment.yaml:26]: use allOf and override example

File: src/components/schemas/adjustment.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    to_account_id:
+      $ref: './fields.yaml#/id_string'
+    settlement_id:
+      $ref: './fields.yaml#/id_string'
+    payment_id:
```
Comment (jayachandhar) [inline src/components/schemas/adjustment.yaml:31]: are we exposing payment_id ?

File: src/components/schemas/reimbursement.yaml [hunk 4]
```diff
@@ -13,22 +13,64 @@ reimbursement_out:
...
       $ref: './fields.yaml#/updated_at'
+    
+    reimbursement_number:
+      type: string
+      description: |
+        A unique, organization wide identifier for a particular reimbursement.<br>
+        This id is always prefixed with `P/`.
+      example: P/2020/11/29
+
+    creator_user_id:
```
Comment (jayachandhar) [inline src/components/schemas/reimbursement.yaml:24]: do we need creator_user_id ? 
we have created_by and updated_by

File: src/components/schemas/settlement.yaml [hunk 5]
```diff
@@ -13,31 +13,38 @@ settlement_out:
...
+    user_account_snapshots:
       type: array
       items:
         type: object
         properties:
-          id:
-            allOf:
-              - $ref: './fields.yaml#/id_string'
+          account_id:
+            $ref: './fields.yaml#/id_string'
```
Comment (jayachandhar) [inline src/components/schemas/settlement.yaml:47]: $ref and example key cannot be used in this way, use allOf

File: src/components/schemas/adjustment.yaml (hunk 3 above)
Comment (sanskarsharma) [reply src/components/schemas/adjustment.yaml:31]: no, removing

File: src/components/schemas/reimbursement.yaml (hunk 4 above)
Comment (sanskarsharma) [reply src/components/schemas/reimbursement.yaml:24]: hmm, couldn't find any mention of these fields in any OpenAPI spec so assumed they might not be external. are they ?

**Pull request merged**